ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -614,6 +614,38 @@ FAN_SS_AL:
   - schedule_run_command
   - control_status
 
+FAN_SS_VSC_SRC:
+  description: "Basic Exhaust fan with start/stop, speed sensor, fail alarm, scheduled run"
+  uses:
+  - lost_power_alarm
+  opt_uses:
+  - filter_alarm
```

Comment:
Hi sir, we have another FAN device that has the same points and an additional filter_alarm, so rather than create another canonical type with filter_alarm, I decided they should share the same type and both can use it.